latest_cov_build.py

```diff
@@ -24,32 +34,33 @@
     url = queue.buildUrl('getLatestArtifact', taskId, 'public/build/{}'.format(name))
     urlretrieve(url, os.path.join('tools', name))
 
-# Extracting coverage build artifact
-with tarfile.open('tools/target.tar.bz2', 'r:bz2') as tar:
-    tar.extractall(path='tools')
-os.remove('tools/target.tar.bz2')
-
 # Geckodriver download
 # OS information for correct geckodriver version
 bitness = platform.architecture()[0]
-latest_version = requests.get('https://api.github.com/repos/mozilla/geckodriver/releases/latest')
-data = latest_version.json()
-tag_name = data['tag_name']
-base_url = 'https://github.com/mozilla/geckodriver/releases/download/' + tag_name + '/geckodriver-' + tag_name + '-'
+
+gecko_url = get_url('https://api.github.com/repos/mozilla/geckodriver/releases/latest',
```

Comment:
`gecko` should be `geckodriver`.
`gecko` is a separate thing, so using `gecko_url` can be confusing (we are not downloading gecko, we are downloading geckodriver).